src/index.ts

```diff
@@ -139,14 +139,16 @@ interface Route {
 	try(path: string): string | undefined
 }
 
-interface Response extends ServerResponse {
+export interface Response extends ServerResponse {
 	locals: { [key: string]: any }
 	log: () => this
 	typeLen: (type: string, len: number) => this
 	send: (type: string, body: string | Buffer) => this
 	format: (map: { [key: string]: () => void }) => this
 	status: (code: number) => this
 	redirect: (location: string) => this
+	etag: (stats: string | Buffer | etag.StatsLike) => this | true
+	cacheControl: (age: number) => this
```

Comment:
As `Cache-Control` header accepts a wide variety of values, it's better to name this method's argument `maxAge`, which is more descriptive.

```suggestion
	cacheControl: (maxAge: number) => this
```